app/push-sw.js

```diff
@@ -72,8 +72,18 @@ function notifyClient(obj) {
  * @return {Promise} Promise resolved once the notification is showed.
  */
 function showNotification(obj) {
-  const title = `Reminder for ${obj.recipient}`;
-  const body = obj.message;
+  let title;
+  if (!obj.recipients.length) {
+    title = 'Reminder';
+  } else if (obj.recipients.length === 1) {
+    title = `Reminder for ${obj.recipients[0]}`;
+  } else {
+    const firstRecipients = obj.recipients.slice(0, -1).join(', ');
+    const lastRecipient = obj.recipients[obj.recipients.length - 1];
+    title = `Reminder for ${firstRecipients}, and ${lastRecipient}`;
```

Comment:
I saw your comment about the Oxford comma. First of all it's not standard, some people use it, other don't. It's not an absolute rule. Second of all, it's only for list of 3 items or more. In this case, we'll still have the comma for 2 items. So I'd suggest to remove it for now.
In the future, I'm thinking of using something like https://github.com/twitter/twitter-cldr-js/blob/master/lib/assets/javascripts/twitter_cldr/en.js#L3188 to generate internationalised list of items.